it/uk/gov/hmrc/minorentityidentificationfrontend/connectors/CreateBusinessVerificationJourneyConnectorISpec.scala

```diff
@@ -17,91 +17,181 @@
 package uk.gov.hmrc.minorentityidentificationfrontend.connectors
```

Comment:
This is a pre-existing issue, but we don't fully test the handling of the response in that there is no test for a response with a status other than CREATED, NOT_FOUND or FORBIDDEN.